_posts/platform/internals/stacks/2000-01-01-scalingo-14-stack.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Scalingo-14 Stack
+nav: Scalingo-14
+modified_at: 2019-12-15 00:00:00
+index: 2
+---
+
+{% warning %}
+  Scalingo-14 support has ended on december. If you're still using this stack, you [must upgrade]({% post_url platform/internals/stacks/2000-01-01-stacks %}).
+{% endwarning %}
+
+This article describes the Scalingo-14 stack, based on Ubuntu 14.04. [What is a stack?]({% post_url platform/internals/stacks/2000-01-01-stacks %}).
+
+## Support period
+
+Scalingo-14 is based on Ubuntu 14.04. It will be supported through April 2023. Learn more about Scalingo’s stack update policy.
```

Comment:
It seems wrong

Reply:
@EtienneM indeed. I don't know the support period. If you know it, can you add it?